1.setup_TD-VAE/scripts/0.setup_TD-VAE.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[8]:
```

Comment:
Just for my understanding - is it important for code cell order to begin at 1 or just for it to be run sequentially from any starting number?

Reply:
Yes, I will rerun these!